### PR TITLE
Adding Instagram media bundle

### PR DESCRIPTION
**Pull request by vijaynandwani**

### CODE REVIEW

File: config/install/media_entity.bundle.instagram.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media_entity_instagram
+id: instagram
+label: Instagram
```
Comment (paranojik) [inline config/install/media_entity.bundle.instagram.yml]: "Instagram post"

File: config/install/field.field.media.instagram.field_instagram_shortcode.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+dependencies:
+  config:
+    - field.storage.media.field_instagram_shortcode
+    - media_entity.bundle.instagram
+id: media.instagram.field_instagram_shortcode
+field_name: field_instagram_shortcode
+entity_type: media
+bundle: instagram
+label: 'Instagram Shortcode'
+description: ''
```
Comment (slashrsm) [inline config/install/field.field.media.instagram.field_instagram_shortcode.yml]: Descriptions.

Comment (vijaynandwani) [inline config/install/field.field.media.instagram.field_instagram_shortcode.yml]: I'll add the descriptions but it won't be visible. Shortcode field is hidden in form display and the url field comes with its own description.

Comment (slashrsm) [inline config/install/field.field.media.instagram.field_instagram_shortcode.yml]: It is still valuable for site builders. Remember, this module is also an example of how things should be implemented. Having nice descriptions will help us to reach that goal as they are still visible in the config UI.

Comment (vijaynandwani) [inline config/install/field.field.media.instagram.field_instagram_shortcode.yml]: I understand. That's right.

**Pull request merged**